be/src/Project/Services/Catalog/Catalog.Api/Services/Categories/CategoriesService.cs

```diff
@@ -51,7 +52,18 @@ public async Task<PagedResults<IEnumerable<CategoryServiceModel>>> GetAsync(GetC
 
             categories = categories.ApplySort(model.OrderBy);
 
-            var pagedResults = categories.PagedIndex(new Pagination(categories.Count(), model.ItemsPerPage), model.PageIndex);
+            var pagedResults = categories.PagedIndex(new Pagination(Constants.EmptyTotal, Constants.MaxItemsPerPageLimit), Constants.DefaultPageIndex);
```

Comment:
**suggestion:** please replace with the following everywhere:

`PagedResults<IEnumerable<Category>> pagedResults;`

Reply:
Hi @dawiddworak88 as variable?